sync/projectutil.py

```diff
@@ -34,7 +36,11 @@ def get(self, *subcommand, **opts):
         assert subcommand and len(subcommand)
         command = [os.path.join(self.path, self.name)] + list(subcommand)
         logger.info("Running command:\n %s" % " ".join(command))
-        return subprocess.check_output(command, cwd=self.path, **opts)
+        try:
+            return subprocess.check_output(command, cwd=self.path, **opts)
+        except subprocess.CalledProcessError as e:
+            newrelic.agent.record_exception(params={"mach_output": e.output})
```

Comment:
This isn't always `mach`. Also I'm worried that the exception is going to end up being recorded twice, unless it's always caught and replaced by an `AbortError`.

Reply:
Ahh yes, forgot I was changing the overall command class. I think replacing it with an AbortError may be the way to go, will adjust commit.